src/som/interpreter/ast/nodes/dispatch.py

```diff
@@ -77,6 +77,9 @@ def __init__(self, rcvr_class, method, next_entry):
         _AbstractDispatchNode.__init__(self, rcvr_class, next_entry)
         self._cached_method = method
 
+    def get_cached_method(self):
```

Comment:
Instead of making the method accessible, should just the node be returned from the lookup method?

This might also give us the option to unify the handling of the does-not-understand case, which is implemented in the `GenericDispatchNode` already.

Reply:
Right, I'd left a comment about it that I removed because pylint would have complained about its 80 columns rule. The reason I returned the method was to ensure the function always returned a method, to avoid returning either a `CachedDispatchNode` or a method directly (in the generic lookup). But I might as well change it and introduce the `GenericDispatchNode` for that DNU case, it does make more sense.